lib/client/checks/utils.ts

```diff
@@ -7,3 +7,10 @@ export const splitStringIntoLines = (
   const lines = inputString.match(regex);
   return lines.join(`\n${linePrefix}`);
 };
+
+export const isUuid = (testValue) => {
+  const regex = new RegExp(
+    `^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$`,
+  );
+  return testValue.match(regex);
```

Comment:
if we bump `uuid` package to 8.3.0 version, we can use `validate()` method: https://github.com/uuidjs/uuid/blob/v8.3.0/src/validate.js#L3